apps/vaults/components/details/actions/QuickActionsButtons.tsx

```diff
@@ -95,7 +95,7 @@ function	VaultDetailsQuickActionsButtons(): ReactElement {
 		toAddress(actionParams?.selectedOptionFrom?.value) === ETH_TOKEN_ADDRESS
 		&& toAddress(actionParams?.selectedOptionTo?.value) === YVWETH_ADDRESS
 	);
-	const hasAllowanceSet = actionParams?.amount.raw.gt(formatBN(allowanceFrom?.raw));
+	const hasAllowanceSet = actionParams?.amount.raw.gt(toBigInt(allowanceFrom?.raw));
```

Comment:
It would be great to get rid of these parsing functions and just have `allowanceFrom?.raw` a BigInt from the get-go, perhaps coercing with Zod. It would clear the codebase quite a bit